src/App.tsx

```diff
@@ -9,39 +9,29 @@ import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
-import { getUserPosts } from './api/posts';
-import { User } from './types/User';
-import { Post } from './types/Post';
+// import { getUsers } from './api/users';
+// import { set } from './features/users';
```

Comment:
These import statements are commented out. If they are not needed, consider removing them to keep the code clean. If they are needed, ensure they are correctly implemented and used in the code.